js-miniapp-bridge/test/test.spec.ts

```diff
@@ -67,3 +69,30 @@ describe('Test Mini App Bridge execErrorCallback is called with no error message
     bridge.MiniAppBridge.prototype.execErrorCallback(callback.id, '');
   });
 });
+
+describe('Test Mini App Bridge execSuccessCallback is called with valid ad response', () => {
+  it('will return success promise and typecast the response JSON string successfully', () => {
```

Comment:
When there is an `and` -> it reflects a possibility to break down the test case for your consideration.